src/app/shared/services/user.service.ts

```diff
@@ -40,11 +40,12 @@ constructor(
   }
 
   getCurrentUser() {
-    this.apollo.query({
+    this.apollo.watchQuery({
```

Comment:
@preethijayanthi Don't make this as watch query, it'll affect the entire app by triggering events on every update cycle

Reply:
But we want to update bro